core/storage/topic/gae_models.py

```diff
@@ -456,6 +447,8 @@ def migrate_model(cls, old_user_id, new_user_id):
             model.manager_ids = [
                 new_user_id if manager_id == old_user_id else manager_id
                 for manager_id in model.manager_ids]
+            # These will be set by the AddAllUserIdsVerificationJob.
+            model.all_user_ids = []
```

Comment:
This is wrong, right? We should not be modifying the TopicModel...

Reply:
Done.